frontend/src/pages/badge.tsx

```diff
@@ -0,0 +1,166 @@
+      fill="black"
+    />
+  </svg>
+);
+
+const RightBackground = ({ textColor, ...props }) => (
+  <svg
+    width="74"
+    height="425"
+    viewBox="0 0 74 425"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    <path
+      d="M91 12.808V433.177C91 433.246 90.9303 433.28 90.8607 433.28H20.9755C46.9779 405.985 46.8647 351.922 20.6185 324.835C-5.73206 297.643 -5.73206 243.226 20.6185 216C46.9691 188.808 46.9691 134.391 20.6185 107.165C-5.73206 79.9732 -5.73206 25.5557 20.6185 -1.67042C46.6209 -28.4724 43.6786 -81.7995 18.2768 -109.38C61.5329 -85.8497 91 -39.4286 91 12.808Z"
+      fill={textColor}
+    />
+  </svg>
+);
+
+const Badge = () => {
+  const router = useRouter()
+
+  const [name, setName] = useState("unset")
+  const [tagline, setTagline] = useState("unset")
+  const [variant, setVariant] = useState("participant")
+
+  useEffect(() => {
+    // @ts-ignore
+    window.changeBadgeData = ({name, tagline, variant}) => {
+      setName(name)
+      setTagline(tagline)
+      setVariant(variant)
+    }
+  }, []);
+
+  // const name = router.query.name;
+  // const tagline = router.query.tagline;
+  // const variant = router.query.variant;
+
+  const isSpeaker = variant === 'speaker';
+  const isParticipant = variant === 'participant';
+  const isKeynoter = variant === 'keynoter';
+  const isStaff = variant === 'staff';
+
+  let textColor = '#F17A5D'
+
+  if (isSpeaker) {
+    textColor = '#F17A5D'
+  } else if (isKeynoter) {
+    textColor = '#9373B0'
+  } else if (isParticipant) {
+    textColor = '#36B39F'
+  } else if (isStaff) {
+    textColor = '#DC9BC4'
+  }
+
+  console.log("router", router.query)
+
+  return (
+    <Box
+      sx={{
+        // width: '550px',
+        // height: '850px',
+        // width: "275px",
+        // height: "425px",
+        width: "10cm",
+        height: "10cm",
+        backgroundColor: "#FCE8DE",
+        position: "relative",
+        overflow: "hidden",
+      }}
+    >
+      <Box
+        sx={{
+          mt: "17px",
+
+          ml: "13px",
+          mr: "70px",
+        }}
+      >
+        <Logo
+          sx={{
+            width: "100px",
+          }}
+        />
+
+        <Text
+          sx={{
+            fontSize: "36px",
```

Comment:
maybe we should use `pt` for print?

Reply:
I used this to convert everything to points: https://simplecss.eu/pxtoems.html